src/components/pages/settings/LayersManager.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import LayersForm from './LayersFormContainer';
+import LayersSwitcher from './LayersSwitcherContainer';
+
+class LayersManager extends React.Component {  
+  render() {
+    return (
+      <div className="c-layers-manager">
```

Comment:
This is interesting, and also not particularly important, but I generally use .l-class as opposed to .c-class when something contains lots of other components. Layout classes handle positioning on the page etc, and .c-classes handle the finer details. Good to follow the same format through our projects so would be good to decide on this.